src/room/LobbyView.tsx

```diff
@@ -193,6 +193,13 @@ export const LobbyView: FC<Props> = ({
 
   useTrackProcessorSync(videoTrack);
 
+  const onEnterCall = useCallback(() => {
+    setWaiting(true);
+    void onEnter().finally(() => setWaiting(false));
+  }, [onEnter]);
+
+  const [waiting, setWaiting] = useState(waitingForInvite);
```

Comment:
I know that `waitingForInvite` never actually gets updated in practice without fully unmounting and remounting the lobby view, but just to avoid a future surprise: could we have this component react to its updates?

```suggestion
  const [waitingToEnter, setWaitingToEnter] = useState(false);
  const waiting = waitingForInvite || waitingToEnter;
```